inductiva/__init__.py

```diff
@@ -5,6 +5,8 @@
 
 import absl
 
+from inductiva.api.methods import validate_api_key
+
 from . import api
```

Comment:
you already import the api module 2 lines below.
You can use it.

```suggestion
from . import api
```